From 41f77396467d67b08cfb782f1e43a99d52cd58f0 Mon Sep 17 00:00:00 2001
From: Junping Du <junping_du@apache.org>
Date: Sun, 8 Jan 2017 15:31:07 -0800
Subject: [PATCH 2247/2848] HADOOP-13958. Bump up release year to 2017.
 Contributed by Junping Du.

(cherry picked from commit 94f1d06f205eb5679475569b932eb2792f53d891)
(cherry picked from commit 7b5b57e5b2b5d33fd73ddf65d3a16f703801953a)
(cherry picked from commit 8f2fa65a8eaea32c2bbe1ddb03011c2518775b77)

Change-Id: I31c79dba95db8fb0ee43fdb470a3e71978814f70
---
 hadoop-project/pom.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index 6d057b0..b78777b 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -31,7 +31,7 @@
 
   <properties>
     <!-- Set the Release year during release -->
-    <release-year>2016</release-year>
+    <release-year>2017</release-year>
 
     <failIfNoTests>false</failIfNoTests>
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
-- 
1.7.9.5

